src/components/LeftSideBar.js

```diff
@@ -169,7 +175,7 @@ class LeftSideBar extends Component {
         );
       } else {
         return (
-          <nav className="scrollLeftBar col-md-2 d-none d-md-block bg-dark sidebar">
+          <nav className="fixed-left scrollLeftBar col-md-2 d-md-block bg-dark sidebar ">
```

Comment:
Does `fixed-left` do anything? I cant find it anywhere in the code base. Also not sure if display the left side bar on top of the table is a mobile responsive solution. Can you make it slide in and out, maybe?

Reply:
What, animations?? Uuhhh...haven't made any animations before.. And why not have it fold up? It seems functional on my screen. Where would you recommend the button be instead?

Reply:
If you press the carrot, it hides the left side bar, just as before.